modules/consul/examples_test.go

```diff
@@ -0,0 +1,39 @@
+package consul_test
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/consul"
+)
+
+func ExampleRunContainer() {
+	// runconsulContainer {
```

Comment:
Let's match the placeholder used in the docs (https://github.com/testcontainers/testcontainers-go/pull/2194/files#diff-722ef8837b8f820bf7c2f317340222a02ce1665a06b5f551bb065e6b5c5b348cR20)
```suggestion
	// runConsulContainer {
```